doc/cephfs/kernel.rst

```diff
@@ -9,16 +9,20 @@ monitor host name(s) into IP address(es) for you. For example::
 	sudo mkdir /mnt/mycephfs
 	sudo mount -t ceph 192.168.0.1:6789:/ /mnt/mycephfs
 
-To mount the Ceph file system with ``cephx`` authentication enabled, you must
-specify a user name and a secret. ::
+To mount the Ceph file system with ``cephx`` authentication enabled, the kernel
+must authenticate with the cluster. The default ``name=`` option is ``guest``.
+The secret can be specified with the ``secret=`` option. ::
 
 	sudo mount -t ceph 192.168.0.1:6789:/ /mnt/mycephfs -o name=admin,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
 
 The foregoing usage leaves the secret in the Bash history. A more secure
 approach reads the secret from a file. For example::
 
 	sudo mount -t ceph 192.168.0.1:6789:/ /mnt/mycephfs -o name=admin,secretfile=/etc/ceph/admin.secret
-	
+
+If a secret is not specified in the options, then the mount helper will attempt
+to find it in a ceph keyring file.
```

Comment:
I think we could do a better job suggesting this new way is the preferred approach. Maybe reorder and suggest that secret= secrefile= are for finer control or mounts without a keyring file.

Reply:
Good point. I cleaned the document up a bit. Hopefully this will be a bit clearer.